vigranumpy/test/CMakeLists.txt

```diff
@@ -153,7 +153,7 @@ IF(NOT PYTHON_NOSETESTS_NOT_FOUND)
     # variable CMAKE_CFG_INTDIR contains a dot '.' for a Windows nmake build, or
     #                           '$(OutDir)' for a VisualStudio build (OutDir will be
     #                           set by VS at build time)
-    IF(NOT CMAKE_CFG_INTDIR STREQUAL ".")
+    IF(NOT CMAKE_GENERATOR MATCHES "^Visual Studio")
```

Comment:
Shouldn't the condition read
```
IF(CMAKE_GENERATOR MATCHES "^Visual Studio")
```
to reproduce the old behavior? (`IF(NOT CMAKE_CFG_INTDIR STREQUAL ".")` means "not nmake", whereas `IF(NOT CMAKE_GENERATOR MATCHES "^Visual Studio")` means "not VS").